src/components/things/names.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Class used to map keys to string literals
```

Comment:
This is a vague comment. How about something like `Stores known names of Things`?